src/board/index.ts

```diff
@@ -74,12 +77,24 @@ export class Board {
     return result ?? id;
   };
 
+  private initialModulePromise: Promise<ModuleWrapper> | undefined;
+  /**
+   * Defined during start().
+   */
+  private modulePromise: Promise<ModuleWrapper> | undefined;
+  /**
+   * Flag to trigger a reset after start finishes.
+   */
+  private resetWhenDone: boolean = false;
+
   constructor(
-    public operations: WebAssemblyOperations,
     private notifications: Notifications,
     private fs: FileSystem,
     private svg: SVGElement
   ) {
+    // We start this loading here so as not to delay loading WASM until start().
+    this.initialModulePromise = this.createModule();
```

Comment:
I think this dance is pointless now we load and compile the WASM up-front. The intention was just to avoid delaying WASM download/compile until start() was called.

Reply:
b8e0b53728a7e858bb9b2328bda5f9357f638ebd